mlir/include/mlir/Transforms/DialectConversion.h

```diff
@@ -1130,10 +1130,9 @@ struct ConversionConfig {
   /// builtin.unrealized_conversion_cast ops. The conversion process fails if
   /// at least one materialization could not be built.
   ///
-  /// If set to "false", the dialect conversion does not does not build any
-  /// custom materializations and instead inserts
-  /// builtin.unrealized_conversion_cast ops to ensure that the resulting IR
-  /// is valid.
+  /// If set to "false", the dialect conversion does not build any custom
+  /// materializations and instead inserts builtin.unrealized_conversion_cast
```

Comment:
Nit: maybe wrap the op name in some form of quotation marks?
```suggestion
  /// materializations and instead inserts 'builtin.unrealized_conversion_cast'
```